src/factories/createPdf.ts

```diff
@@ -37,44 +34,11 @@ export type Template<T extends TemplateSchema> = {
   stories: Record<string, Story<TemplateData<T>>>;
 };
 
+registerFonts();
+
 // biome-ignore lint/complexity/noBannedTypes: <explanation>
 const TemplateContext = createContext<{}>({});
 
-const registerFonts = () => {
-  const baseDirectory = path.resolve(__dirname, "..", "fonts", "Inter");
```

Comment:
The ﻿`node:path` module is not compatible with browser environments. We'll need to find a cross-platform solution that works on both the server and the client.

Reply:
To keep things moving, I'll submit a separate PR for #3.